courses/views.py

```diff
@@ -78,3 +80,23 @@ def get_queryset(self):
 class RequirementList(generics.ListAPIView, BaseCourseMixin):
     serializer_class = RequirementListSerializer
     queryset = Requirement.objects.all()
+
+
+class UserView(generics.RetrieveAPIView, generics.UpdateAPIView):
+    serializer_class = UserSerializer
+    permission_classes = [IsAuthenticated]
+
+    def get_queryset(self):
+        return get_user_model().objects.all().prefetch_related('profile')
```

Comment:
Can we do `filter(pk=self.request.user.pk)` instead of `all()`? Just to be sure we're not exposing access to all users

Reply:
Done